From eb38e0d4ef39459554d112270fd8d2a73ad2d743 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sat, 4 Jan 2025 23:41:53 -0500
Subject: [PATCH 1/2] Use conda-forge's plutovg

---
 .../hello_imgui_build_lib.cmake               | 23 +++++++++++++++----
 1 file changed, 18 insertions(+), 5 deletions(-)

diff --git a/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake b/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake
index 987e64f..9a28b19 100644
--- a/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake
+++ b/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake
@@ -279,11 +279,24 @@ function(him_build_imgui)
             _him_checkout_imgui_submodule_if_needed()
             _him_do_build_imgui()
         endif()
-        if (HELLOIMGUI_USE_FREETYPE)
-            _him_add_freetype_to_imgui()
-            if (HELLOIMGUI_USE_FREETYPE_PLUTOSVG)
-                _him_add_freetype_plutosvg_to_imgui()
-            endif()
+        if (WIN32)
+            # Using pkgconfig on windows just didn't work.
+            # Provide the full directories, hardcode this.
+            link_directories(imgui $ENV{LIBRARY_LIB})
+            target_link_libraries(imgui PUBLIC $ENV{LIBRARY_LIB}/plutovg.lib)
+            target_link_libraries(imgui PUBLIC $ENV{LIBRARY_LIB}/plutosvg.lib)
+            target_include_directories(imgui PRIVATE $ENV{LIBRARY_INC}/plutovg)
+            target_include_directories(imgui PRIVATE $ENV{LIBRARY_INC}/plutosvg)
+        else()
+            find_package(PkgConfig)
+            # Add the plutosvg include directory from the prefix
+            pkg_search_module(PLUTOVG REQUIRED plutovg)
+            target_link_libraries(imgui PRIVATE ${PLUTOVG_LIBRARIES})
+            target_include_directories(imgui PRIVATE ${PLUTOVG_INCLUDE_DIRS})
+
+            pkg_search_module(PLUTOSVG REQUIRED plutosvg)
+            target_link_libraries(imgui PRIVATE ${PLUTOSVG_LIBRARIES})
+            target_include_directories(imgui PRIVATE ${PLUTOSVG_INCLUDE_DIRS})
         endif()
     endif()
 endfunction()
-- 
2.47.1

