From 74981f9796aeb3fba1c9d817d56ded642f698694 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sat, 21 Mar 2026 19:19:10 -0400
Subject: [PATCH 2/2] Find glfw correctly for conda-forge

---
 bindings/imgui_bundle/__init__.py | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/bindings/imgui_bundle/__init__.py b/bindings/imgui_bundle/__init__.py
index e54b1a6..c84fb38 100644
--- a/bindings/imgui_bundle/__init__.py
+++ b/bindings/imgui_bundle/__init__.py
@@ -230,15 +230,6 @@ if has_submodule("immvision"):
     __all__.extend(["imgui_fig"])
 
 
-# Glfw setup:
-# By importing imgui_bundle.glfw_utils, we make sure that glfw provided by pip will use our glfw dynamic library.
-# (imgui_bundle.glfw_utils will call _set_glfw_pip_search_path automatically)
-if has_submodule("with_glfw"):
-    from imgui_bundle._glfw_set_search_path import _glfw_set_search_path
-
-    _glfw_set_search_path()
-    from imgui_bundle import glfw_utils as glfw_utils  # noqa: E402
-
 #
 # Pyodide: patch hello_imgui.run and immapp.run to work with Pyodide
 #
-- 
2.43.0

